server/controllers/stock/core.js

```diff
@@ -402,14 +408,38 @@ function stockManagementProcess(inventories) {
     Q = inventory.quantity; // the quantity
```

Comment:
We can also mention that this quantity is the available quantity of inventory for a given depot, the implementation is in : `/server/controllers/stock/core.js`

Reply:
Great idea.  Thank you!